### PR TITLE
Add autocomplete icon

### LINKED ISSUE #46: Add autocomplete icon
Since v5.24 slash commands can provide an `svg` via  `AutocompleteIconData` in `model.Command` that is then shown in the autocomplete list. This ticket is about adding the icon in `assets` to the slash command.

Include screenhots from a dark and a bright theme in the PR description, as there might be the need for a separate icon with a white background for autocomplete.

Similar PRs:
For GitHub plugin https://github.com/mattermost/mattermost-plugin-github/pull/359
For Jira plugin https://github.com/mattermost/mattermost-plugin-jira/pull/653

### PR DESCRIPTION
**Pull request by ByeongsuPark**
#### Summary

Screenshot_2020-11-12_15-11-17
Screenshot_2020-11-12_15-11-51

Add autocomplete icon

#### Ticket Link

Fixes #46

### CODE REVIEW

File: server/command.go [hunk 1]
```diff
@@ -94,3 +97,11 @@ func getAutoCompleteData() *model.AutocompleteData {
 	aws.AddCommand(listTopics)
 	return aws
 }
+
+func (p *Plugin) getAutoCompleteIconData() string {
+	iconData, err := command.GetIconData(p.API, "assets/icon.svg")
+	if err != nil {
+		p.API.LogError("Failed to get icon data")
```
Comment (hanzei) [inline server/command.go:104]: Please log the error

**Review by hanzei — APPROVED** [review]
Thanks for the contribution :+1: 

One nit pick

File: server/command.go [hunk 2]
```diff
@@ -94,3 +97,11 @@ func getAutoCompleteData() *model.AutocompleteData {
 	aws.AddCommand(listTopics)
 	return aws
 }
+
+func (p *Plugin) getAutoCompleteIconData() string {
+	iconData, err := command.GetIconData(p.API, "assets/icon.svg")
+	if err != nil {
+		errors.Wrap(err, "Failed to get icon data")
```
Comment (hanzei) [inline server/command.go:104]: Did you meant to return the error here?

Comment (ByeongsuPark) [reply server/command.go:104]: Yes, that's right. Is there anything to improve on this code? I just think that error handling in registerCommands() function can be better

Comment (hanzei) [reply server/command.go:104]: Maybe a separate function is not even needed. You could use something like: https://github.com/mattermost/mattermost-plugin-github/blob/88aee8396d6dc238fa5507804f3dd4aa751d338a/server/plugin/command.go#L72-L76

Comment (ByeongsuPark) [reply server/command.go]: It looks good. 
Thank you for letting me know the better way. I refactored the code so that a separate function is not needed.

**Review by hanzei — APPROVED** [review]
Nice work :+1:

**Review by mickmister — APPROVED** [review]
LGTM :+1: Thanks @ByeongsuPark!

**Comment by hanzei** [comment]
Tested on a bright and dark theme. No issues found.

**Pull request merged**